### PR TITLE
docs(READEME): update kind version

### PR DESCRIPTION
**Pull request by zaunist**
Currently, we release 0.12.0, so we should update the `READEME.md`

### CODE REVIEW

**Comment by zaunist** [comment]
Hi, community.

I noticed our `site` docs need be update too, do I update it together in this PR or in a new PR?

**Comment by tao12345666333** [comment]
Please complete the update in this PR, thanks.

**Comment by zaunist** [comment]
I have updated the docs, please take a look, thanks.

**Comment by aojea** [comment]
/ok-to-test
/lgtm
/approve
Thanks

File: site/content/docs/user/configuration.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ networking:
 {{< /codeFromInline >}}
 
 ##### Dual Stack clusters
-You can run dual stack clusters using `kind` 0.11+, on kubernetes versions 1.20+.
+You can run dual stack clusters using `kind` 0.12+, on kubernetes versions 1.20+.
```
Comment (BenTheElder) [inline site/content/docs/user/configuration.md:143]: this change is wrong, this feature was not introduced in v0.12.0

Comment (aojea) [reply site/content/docs/user/configuration.md:143]: I overlooked this , sorry,

Ben is right , is not about being semantically correct , this section talks about the dual stack feature in kind, that is 0.11 ... just revert this change to 0.11+

Comment (zaunist) [reply site/content/docs/user/configuration.md:143]: Sorrry, I will revert this change later.

**Pull request merged**